src/main/scala/edu/ie3/simona/io/result/ResultEntityCsvSink.scala

```diff
@@ -197,28 +149,31 @@ object ResultEntityCsvSink {
     *   mapping of the class that should be processed
     * @param delimiter
     *   the delimiter of the .csv-file
-    * @param bufferSize
-    *   maximum size of the buffer of this sink
-    * @param materializer
-    *   the materializer to be used by the stream that writes the output file
     * @return
     *   instance of [[ResultEntityCsvSink]] to be used to write results
     */
   def apply(
       outfileName: String,
       resultEntityProcessor: ResultEntityProcessor,
       compressOutputFiles: Boolean,
-      delimiter: String = ",",
-      bufferSize: Int = 50
-  )(implicit materializer: Materializer): Future[ResultEntityCsvSink] = {
+      delimiter: String = ","
+  ): ResultEntityCsvSink = {
+
+    val file = new File(outfileName)
+    val existedBefore = file.exists()
+
+    val writer = new BufferedWriter(new FileWriter(file, existedBefore), 32768)
```

Comment:
It's `2^15`, we could also adjust the size. Haven't spent that much time thinking about what could be a reasonable buffer size. Choosing a rather large size is fine since the buffer is supposed to be flushed in the end anyways.